docs/cli-tool/kubectl-plugin.md

```diff
@@ -82,6 +94,17 @@ $ kubectl kruise rollout status statefulsets.apps.kruise.io/sts2
 # approve a kruise rollout resource named "rollout-demo" in "ns-demo" namespace
 $ kubectl-kruise rollout approve rollout-demo -n ns-demo
 ```
+### describe
+
+Perform a deep inspection of any Kruise object (including Rollouts).
+
+```bash
+# Inspect the “rollouts-demo” Rollout in namespace default:
+kubectl kruise describe rollout rollouts-demo -n default
+
+# You can also describe other kruise objects, for example:
+kubectl kruise describe cloneset my-cloneset -n default
```

Comment:
current the enhanced version of "describe" is only supported for rollout not for any other resources